### PR TITLE
USWDS-Site - Changelog:  Character count visual cues [#5908]

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Adds changelog for uswds/uswds#5908

## Related PR

[USWDS - Character count: Enhance visual cue when limit is exceeded #5908](https://github.com/uswds/uswds/pull/5908)

## Preview link

[Preview →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-changelog-5908/components/character-count/#latest-updates)

### CODE REVIEW

**Comment by mahoneycm** [comment]
@cathybaptista Merge conflict resolved! I also went ahead and removed the USWDS version number for now 👍

File: _data/changelogs/component-character-count.yml [hunk 1]
```diff
@@ -2,6 +2,13 @@ title: Character count
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Enhanced visual cue when maxlength is exceeded.
+    summaryAdditional:  Now, the visual state of the character count component will reflect its validation state.
```
Comment (amyleadem) [inline _data/changelogs/component-character-count.yml:7]: **suggestion:** Recommend that we specifically refer to the error state here for clarity. Here is a possible suggestion (please feel free to re-write):

```suggestion
    summary: Enhanced visual cue when `maxlength` is exceeded.
    summaryAdditional:  Now, the component uses standard USWDS error styles to visually enhance the error state.
```

File: _data/changelogs/component-character-count.yml [hunk 2]
```diff
@@ -2,6 +2,13 @@ title: Character count
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Enhanced visual cue when `maxlength` is exceeded.
+    summaryAdditional:  Now, the component uses standard USWDS error styles to visually enhance the error state.
+    affectsJavascript: true
```
Comment (amyleadem) [inline _data/changelogs/component-character-count.yml:8]: **question**: Should this include the "styles" tag? The related PR doesn't touch the CSS itself, but does change how the component visually displays. Flagging to hear other thoughts around this. 
```suggestion
    affectsJavascript: true
    affectsStyles: true
```

Comment (mejiaj) [reply _data/changelogs/component-character-count.yml:8]: It should according to changelog README description.

https://github.com/uswds/uswds-site/blob/1781d23208346bde7badb2c441535099675745c4/_data/changelogs/_CHANGELOG-README.md?plain=1#L16

**Review by mejiaj — APPROVED** [review]

**Review by amyleadem — APPROVED** [review]

**Pull request merged**